### ISSUE TITLE
make works no longer, um, works

### ISSUE THREAD

**Issue by kcranston** [OpenTreeOfLife/reference-taxonomy#370]
In the [instructions for deploying a new version of the taxonomy](https://github.com/OpenTreeOfLife/germinator/wiki/Deploying-a-new-taxonomy-version#get-final-ott-ids) the command for making the final version with the new IDs is `make works`. That target no longer exists:

`make: *** No rule to make target 'works'.  Stop.`

**Comment by jar398** [comment]
Is need_ids.json nonzero?

**Comment by jar398** [comment]
oh sorry. looks like you got past that point. There is almost certainly a new target to replace the former 'works'

**Comment by jar398** [comment]
I suspect you just run 'make ott' again, and make sure afterward that need_ids.json says that no new ids are needed.

**Comment by jar398** [comment]
Yes I believe so.  If it didn't write a need_ids.json file then it doesn't need any new ids, it already has all the ones it wants. (if I remember correctly, it's been a while)
